tests/performance/3DLighting/attenuation/Environment.java

```diff
@@ -78,6 +83,9 @@
         pointLight2.setTranslateX(-LIGHT_X_DIST);
         spotLight2.setTranslateX(-LIGHT_X_DIST);
 
```

Comment:
I recommend setting the direction of `directionalLight1` and `directionalLight2` as follows:

```
    directionalLight1.setDirection(new Point3D(-LIGHT_X_DIST, 0, LIGHT_Z_DIST));
    directionalLight2.setDirection(new Point3D(LIGHT_X_DIST, 0, LIGHT_Z_DIST));
```